pkg/deployment/ingester_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/require"
```

Comment:
I think the import order here is not correct

Reply:
it passes CI check. We should perhaps use different tooling to catch this on CI